test/e2e/multikueue/e2e_test.go

```diff
@@ -491,6 +495,56 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 		})
+
+		ginkgo.It("Should run a RayJob on worker if admitted", func() {
+			E2eKuberayTestImage := "rayproject/ray:2.9.0"
+			if runtime.GOOS == "darwin" {
+				E2eKuberayTestImage = "rayproject/ray:2.9.0-py39-aarch64"
+			}
+			fmt.Println(runtime.GOOS, E2eKuberayTestImage)
```

Comment:
```suggestion
			ginkgo.GinkgoLogr.Info(runtime.GOOS, E2eKuberayTestImage)
```
Or maybe remove it?

Reply:
yes, I will remove it, it was for debugging purposes